com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -542,6 +542,15 @@ static void RenderCameraStack(ScriptableRenderContext context, Camera baseCamera
                             if (baseCameraData.xr.enabled)
                                 m_XRSystem.UpdateFromCamera(ref overlayCameraData.xr, overlayCameraData);
 #endif
+
+                            // There is still some native codes that depends on camera properties
+                            // Current know native systems:
+                            // - ScriptableRendererContext.SetupCameraProperties.
+                            // - Screen space camera canvas.
+                            currCamera.targetTexture = baseCamera.targetTexture;
+                            currCamera.targetDisplay = baseCamera.targetDisplay;
+                            currCamera.rect = baseCamera.rect;
```

Comment:
I could actually add conditions if (currCamera.targetDisplay == baseCamera.targetDisplay). However native code inside takes care of it. So I can add it just to avoid scripting cost